pkg/core/pipeline/targets.go

```diff
@@ -38,41 +36,27 @@ func (p *Pipeline) RunTargets() error {
 		target := p.Targets[id]
 		target.Config = p.Config.Targets[id]
 
-		rpt := p.Report.Targets[id]
+		report := p.Report.Targets[id]
 		// Update report name as the target configuration might has been updated (templated values)
-		rpt.Name = target.Config.Name
+		report.Name = target.Config.Name
 
 		err = target.Run(p.Sources[target.Config.SourceID].Output, &p.Options.Target)
-
-		rpt.Result = target.Result
-
 		if err != nil {
-			logrus.Errorf("Something went wrong in target \"%v\" :\n", id)
-			logrus.Errorf("%v\n\n", err)
-
-			isResultIsFailed = true
-
-			p.Targets[id] = target
-			p.Report.Targets[id] = rpt
-			i++
-			continue
-		}
-
-		if strings.Compare(target.Result, result.ATTENTION) == 0 {
-			isResultIsChanged = true
+			p.Report.Result = result.FAILURE
+			return fmt.Errorf("Something went wrong in target \"%v\" :\n%v\n\n", id, err)
```

Comment:
```suggestion
			return fmt.Errorf("something went wrong in target \"%v\" :\n%v\n\n", id, err)
```
An error message shouldn't start with uppercase as it could be embedded in another error message